### PR TITLE
MH-250: Added Featured Content to ModularHistory Homepage

### PR DESCRIPTION
**Pull request by sunzae99**
The Featured Content is currently being implemented using the ModuleUnionCard, which needs some styling. (This can be done later, in a separate ticket)

### CODE REVIEW

File: frontend/pages/index.page.tsx [hunk 1]
```diff
@@ -133,3 +146,79 @@ const AboutModularHistory: FC = () => {
     </Box>
   );
 };
+
+const FeatureContent: FC = () => {
```
[the remark this answers is not in the document]
Comment (sunzae99) [reply frontend/pages/index.page.tsx]: Made changes on the following commit!

File: frontend/pages/index.page.tsx [hunk 2]
```diff
@@ -133,3 +146,79 @@ const AboutModularHistory: FC = () => {
...
+                      </Link>
+                    ))
+                  ) : loading ? (
+                    <Card>
+                      <CardContent>
+                        <Skeleton sx={{ minHeight: 200 }} />
+                      </CardContent>
+                    </Card>
+                  ) : (
+                    <p>There are no modules associated with this date.</p>
```
[the remark this answers is not in the document]
Comment (sunzae99) [reply frontend/pages/index.page.tsx]: screencapture-modularhistory-dev-net-2021-10-30-19_44_08

File: apps/home/api/urls.py [hunk 3]
```diff
@@ -5,5 +5,6 @@
 app_name = 'home'
 
 urlpatterns = [
+    path('feature/', views.FeatureAPIView.as_view()),
```
Comment (iacobfred) [inline apps/home/api/urls.py:8]: _nit_: Let's make this plural: `'features/'`

File: frontend/pages/index.page.tsx [hunk 4]
```diff
@@ -90,6 +89,24 @@ export default function Home() {
...
+        columns={{ xs: 12 }}
+        alignItems={"center"}
+        justifyContent={"center"}
+      >
+        <Grid item xs={12} sm={6} md={6} alignItems="center" justifyContent="center">
+          <Container>
+            <AboutModularHistory />
+          </Container>
+        </Grid>
+        <Grid item xs={12} sm={6} md={6}>
```
Comment (iacobfred) [inline frontend/pages/index.page.tsx:104]: @sunzae99, the featured content should be displayed before the "today in history" content, I think.

File: frontend/pages/index.page.tsx [hunk 5]
```diff
@@ -144,3 +161,76 @@ const AboutModularHistory: FC = () => {
...
+                        <Grid item>
+                          <ModuleUnionCard module={module} key={index} />
+                        </Grid>
+                      </a>
+                    </Link>
+                  ))
+                ) : loading ? (
+                  <Card>
+                    <CardContent>
+                      <Skeleton sx={{ minHeight: 200 }} />
```
Comment (iacobfred) [inline frontend/pages/index.page.tsx:223]: nit: Let's make this `Skeleton` full-width.

**Review by iacobfred — CHANGES_REQUESTED** [review]

**Review by iacobfred — APPROVED** [review]

**Pull request merged**